android/src/main/java/com/rudderstack/android/internal/plugins/PlatformInputsPlugin.kt

```diff
@@ -245,7 +245,7 @@ internal class PlatformInputsPlugin : Plugin, LifecycleListenerPlugin {
             "model" to Build.MODEL,
             "name" to Build.DEVICE,
             "type" to "Android",
-            "adTrackingEnabled" to autoCollectAdvertisingId
+            "adTrackingEnabled" to !_deviceId.isNullOrEmpty()
```

Comment:
Why `adTrackingEnabled` flag is dependent upon the `deviceId` value? Am I missing something?

Reply:
Ideally this `adTrackingEnabled` flag should depend upon `advertsingId`!